target/scripts/startup/setup.d/security/misc.sh

```diff
@@ -263,3 +263,22 @@ function __setup__security__amavis
     fi
   fi
 }
+
+# We can use Sieve to move spam emails to the "Junk" folder.
+function _setup_spam_to_junk
+{
+  if [[ ${MOVE_SPAM_TO_JUNK} -eq 1 ]]
+  then
+    _log 'debug' 'Spam emails will be moved to the Junk folder'
+    sievec /usr/lib/dovecot/sieve-global/after/spam_to_junk.sieve
+    chown dovecot:dovecot /usr/lib/dovecot/sieve-global/after/spam_to_junk.{sieve,svbin}
+
+    if [[ ${ENABLE_SPAMASSASSIN} -eq 1 ]] && [[ ${SPAMASSASSIN_SPAM_TO_INBOX} -eq 0 ]]
+    then
+      _log 'warning' "'SPAMASSASSIN_SPAM_TO_INBOX=0' but it is required to be 1 for 'MOVE_SPAM_TO_JUNK=1' to work"
+    fi
+  else
+    _log 'debug' 'Spam emails will not be moved to the Junk folder'
+    rm /usr/lib/dovecot/sieve-global/after/spam_to_junk.sieve
```

Comment:
Is it not relevant to also `rm` the `.svbin`? This seemed to be done previously.

```suggestion
    rm /usr/lib/dovecot/sieve-global/after/spam_to_junk.{sieve,svbin}
```

Reply:
The `.svbin` file does not exists, because it is created only in case the `if` evaluated to true and `sievec` is called.